src/Bicep.Core.UnitTests/FileSystem/FileResolverTests.cs

```diff
@@ -26,21 +26,21 @@ public void TryResolveModulePath_should_return_expected_results(string parentFil
         {
             if (CompilerFlags.IsWindowsBuild)
             {
-                yield return new [] { "file:///C:/abc/file.bicep", "def/ghi.bicep", "file:///C:/abc/def/ghi.bicep", };
-                yield return new [] { "file:///C:/abc/file.bicep", "./def/ghi.bicep", "file:///C:/abc/def/ghi.bicep", };
-                yield return new [] { "file:///C:/abc/file.bicep", "../ghi.bicep", "file:///C:/ghi.bicep", }; // directory is resolved relative to parent path
-                yield return new [] { "file:///C:/abc/file.bicep", "./def/../ghi.bicep", "file:///C:/abc/ghi.bicep", }; // directory is resolved
-                yield return new [] { "file:///D:/abc/file.bicep", "/def.bicep", "file:///D:/def.bicep", }; // root child path is used as-is
-                yield return new [] { "file:///E:/abc/file.bicep", "/abc/../def/../ghi.bicep", "file:///E:/ghi.bicep", }; // root child path is used as-is with normalization
+                yield return new[] { "file:///C:/abc/file.bicep", "def/ghi.bicep", "file:///C:/abc/def/ghi.bicep", };
```

Comment:
Do you guys have styling requirements?  Both vscode and vs want to reformat the spacing here.  I can redo without format changes if needed.

Reply:
Nothing that's been formalized or enforced by any analyzers. The re-formatting looks good to me.